cmd/flag/write_file_flags.go

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright © 2020 Infostellar, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flag
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+type WriteFileFlag struct {
+	FileName      string
+	TelemetryFile *os.File
+}
+
+// Add a flag to the command.
+func (f *WriteFileFlag) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&f.FileName, "output-file", "o", "", "[Alpha feature] The file to write packets to. Creates file if it does not exist; appends file if it exists. (default none)")
```

Comment:
Btw there's already an `-o` flag for output format for printing pass/plan details like `list-plans` and `list-passes`, but I'm not sure what's the convention regarding separate commands with same short flags but different meanings

Reply:
how about just remove the short flag `-f` all together since this is an alpha feature?